<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>771</width>
      <height>71</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>741</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutTime">
        <item>
         <widget class="QLabel" name="currentTimeLabel">
          <property name="text">
           <string>Time :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentTimeValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutVoltage">
        <item>
         <widget class="QLabel" name="busVoltageLabel">
          <property name="text">
           <string>Bus Voltage (V) :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentBusVoltageValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutCurrent">
        <item>
         <widget class="QLabel" name="shuntVoltageLabel">
          <property name="text">
           <string>Shunt Voltage (mV) :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentShuntVoltageValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutPower">
        <item>
         <widget class="QLabel" name="loadVoltageLabel">
          <property name="text">
           <string>Load Voltage (V) :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentLoadVoltageValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutEnergy">
        <item>
         <widget class="QLabel" name="currentLabel">
          <property name="text">
           <string>Current (mA) :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentCurrentValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayoutFrequency">
        <item>
         <widget class="QLabel" name="powerLabel">
          <property name="text">
           <string>Power (mW) :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="currentPowerValue">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QProgressBar" name="batteryProgressBar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>520</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QWidget" name="centralwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>771</width>
      <height>400</height>
     </rect>
    </property>
    <widget class="QGroupBox" name="historicalDataGroupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>771</width>
       <height>401</height>
      </rect>
     </property>
     <property name="title">
      <string>Historical Data</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>751</width>
        <height>351</height>
       </rect>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>711</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="historicalHeaderLayout">
        <item>
         <widget class="QLabel" name="historicalTimeHeaderLabel">
          <property name="text">
           <string>Time</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalBusVoltageHeaderLabel">
          <property name="text">
           <string>Bus Voltage (V)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalShuntVoltageHeaderLabel">
          <property name="text">
           <string>Shunt Voltage (mV)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicaLoadVoltageHeaderLabel">
          <property name="text">
           <string>Load Voltage (V)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalECurrentHeaderLabel">
          <property name="text">
           <string>Current (mA)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalPowerHeaderLabel">
          <property name="text">
           <string>Power (mW)</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>711</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="historicalDataLayout1">
        <item>
         <widget class="QLabel" name="historicalTimeLabel1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalBusVoltageLabel1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalShuntVoltageLabel1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalLoadVoltageLabel1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalCurrentLabel1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalPowerLabel1">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>711</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="historicalDataLayout2">
        <item>
         <widget class="QLabel" name="historicalTimeLabel2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalBusVoltageLabel2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalShuntVoltageLabel2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalLoadVoltageLabel2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalCurrentLabel2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalPowerLabel2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>711</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="historicalDataLayout3">
        <item>
         <widget class="QLabel" name="historicalTimeLabel3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalBusVoltageLabel3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalShuntVoltageLabel3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalLoadVoltageLabel3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalCurrentLabel3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalPowerLabel3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>711</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="historicalDataLayout4">
        <item>
         <widget class="QLabel" name="historicalTimeLabel4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalBusVoltageLabel4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalShuntVoltageLabel4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalLoadVoltageLabel4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalCurrentLabel4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalPowerLabel4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>711</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="historicalDataLayout5">
        <item>
         <widget class="QLabel" name="historicalTimeLabel5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalBusVoltageLabel5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalShuntVoltageLabel5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalLoadVoltageLabel5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalCurrentLabel5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalPowerLabel5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>711</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="historicalDataLayout6">
        <item>
         <widget class="QLabel" name="historicalTimeLabel6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalBusVoltageLabel6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalShuntVoltageLabel6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalLoadVoltageLabel6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalCurrentLabel6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalPowerLabel6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>711</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="historicalDataLayout7">
        <item>
         <widget class="QLabel" name="historicalTimeLabel7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalBusVoltageLabel7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalShuntVoltageLabel7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalLoadVoltageLabel7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalCurrentLabel7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalPowerLabel7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>711</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="historicalDataLayout8">
        <item>
         <widget class="QLabel" name="historicalTimeLabel8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalBusVoltageLabel8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalShuntVoltageLabel8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalLoadVoltageLabel8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalCurrentLabel8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalPowerLabel8">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>711</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="historicalDataLayout9">
        <item>
         <widget class="QLabel" name="historicalTimeLabel9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalBusVoltageLabel9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalShuntVoltageLabel9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalLoadVoltageLabel9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalCurrentLabel9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalPowerLabel9">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>711</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="historicalDataLayout10">
        <item>
         <widget class="QLabel" name="historicalTimeLabel10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalBusVoltageLabel10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalShuntVoltageLabel10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalLoadVoltageLabel10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalCurrentLabel10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="historicalPowerLabel10">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Data</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
